openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/penalites.py

```diff
@@ -1,92 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
@fpagnoux : I will do as you tell me.
Should I open a PR or a simple issue (can we attach files to issues ?), something else ... ?